koku/masu/processor/orchestrator.py

```diff
@@ -260,20 +261,49 @@ def start_manifest_processing(  # noqa: C901
                 LOG.info(log_json(tracing_id, msg="download queued", schema=schema_name))
 
         manifest_list = [manifest.get("manifest_id") for manifest in manifest_list]
+        LOG.info(
+            log_json(
+                "start_manifest_processing",
+                msg="created manifest list",
+                report_tasks=report_tasks,
+                summarize_reports=self._summarize_reports,
+                schema=schema_name,
+            )
+        )
         if report_tasks:
             if self._summarize_reports:
                 reports_tasks_queued = True
                 hcs_task = collect_hcs_report_data_from_manifest.s().set(queue=HCS_Q)
                 summary_task = summarize_reports.s(
                     manifest_list=manifest_list, ingress_report_uuid=self.ingress_report_uuid
                 ).set(queue=SUMMARY_QUEUE)
+                LOG.info(
+                    log_json("start_manifest_processing", msg="created summary_task signature", schema=schema_name)
+                )
                 # data source contains fields from applications.extra and metered is the key that gates subs processing.
                 subs_task = extract_subs_data_from_reports.s(data_source.get("metered", "")).set(
                     queue=SUBS_EXTRACTION_QUEUE
                 )
+                LOG.info(log_json("start_manifest_processing", msg="created subs_task signature", schema=schema_name))
                 async_id = chord(report_tasks, group(summary_task, hcs_task, subs_task))()
+                LOG.info(
+                    log_json(
+                        "start_manifest_processing",
+                        msg="created chord with group",
+                        async_id=str(async_id),
+                        schema=schema_name,
+                    )
+                )
             else:
                 async_id = group(report_tasks)()
+                LOG.info(
+                    log_json(
+                        "start_manifest_processing",
+                        msg="group report_tasks",
+                        async_id=str(async_id),
+                        schema=schema_name,
+                    )
+                )
             LOG.info(log_json(tracing_id, msg=f"Manifest Processing Async ID: {async_id}", schema=schema_name))
```

Comment:
This async_id is already logged on line 307, so I don't see the benefit of this addition.

Reply:
One reason it's helpful to include is for filtering. If we wanted to filter by async_id, it's helpful to have it in all the log messages.

Reply:
I removed it.